### PR TITLE
feat(webapp): add table and graph for income

### LINKED ISSUE #47: Report total income
Create a Graph line and table  on the income where the different variables are shown (EOS and USD)

Income = withdraws from eden treasury

- [ ] Total income by election
- [ ] Total income by month
- [ ] Total income by delegate
- [ ] Total income for all elections 
- [ ] Top( 10 or 20 ) delegate by income 
table example

### Graph View
https://recharts.org/en-US/
- charts can have controls 
image

### Table View
https://mui.com/material-ui/react-table/
- presents data in a tabular view
- has sorting and filters

### LINKED ISSUE #51: treasury balance component
Example

```
const fetch = require('node-fetch');
const { JsonRpc } = require('eosjs');

// Instantiate a new JsonRpc object, with the Network Api Uri, and a request object
const rpc = new JsonRpc('https://api.kylin.alohaeos.com/', { fetch });
// Request the balance, passing in the token contract, the account name, and the token symbol
rpc.get_currency_balance('eosio.token', 'eospaceioeos', 'EOS').then((balance) => console.log(balance));
```

### PR DESCRIPTION
**Pull request by warnerHurtado**
### What does this PR do?

- Resolve #47 
- Resolve #51 

### Steps to test

1. run the project
2. go to the sidebar and click on income
3. See the view

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check

### CODE REVIEW

File: webapp/src/routes/IncomeReport/styles.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+export default theme => ({
+  eosPriceContainer: {
+    backgroundColor: 'rgba(0, 194, 191, 0.25)',
+    [theme.breakpoints.up('300')]: {
+      width: '244.4px',
+      height: '77px',
+      // margin: '16px 31.6px 3px 318px',
```
Comment (leisterfrancisco) [inline webapp/src/routes/IncomeReport/styles.js:7]: do we need this comment?

File: webapp/src/routes/IncomeReport/styles.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      padding: '4.2px 9.9px 7.7px 17px',
+      borderRadius: '11.3px'
+    }
+  },
+  eosPriceTitle: {
+    // margin: 0 0 27.1px 99.6px;
```
Comment (leisterfrancisco) [inline webapp/src/routes/IncomeReport/styles.js:13]: do we need this comment?

File: webapp/src/routes/IncomeReport/styles.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    color: '#606060',
+    [theme.breakpoints.up('300')]: {
+      fontSize: '11.3px',
+      lineHeight: 1.37,
+      letterSpacing: '-0.23px',
+      textAlign: 'left'
+    }
+  },
+  eosBalance: {
+    // margin: 15.5px 0.3px 1.5px 47.3px;
```
Comment (leisterfrancisco) [inline webapp/src/routes/IncomeReport/styles.js:23]: do we need this comment?

File: webapp/src/routes/IncomeReport/styles.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    color: '#606060',
+    [theme.breakpoints.up('300')]: {
+      fontSize: '22.6px',
+      fontWeight: 'bold',
+      lineHeight: 1.12,
+      letterSpacing: '-0.45px'
+    }
+  },
+  eosBalanceInDollars: {
+    // margin: 1.5px 0.6px 0 0;
```
Comment (leisterfrancisco) [inline webapp/src/routes/IncomeReport/styles.js:33]: do we need this comment?

File: webapp/src/routes/IncomeReport/styles.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    color: '#606060',
+    [theme.breakpoints.up('300')]: {
+      fontSize: '17px',
+      fontWeight: 'bold',
+      lineHeight: 1.25,
+      letterSpacing: '-0.34px'
+    }
+  },
+  tableTitle: {
+    // margin: 1.5px 0.6px 0 0;
```
Comment (leisterfrancisco) [inline webapp/src/routes/IncomeReport/styles.js:43]: do we need this comment?

File: webapp/src/routes/index.js [hunk 6]
```diff
@@ -1,16 +1,18 @@
 import React, { lazy } from 'react'
 
 import {
-  Grid as GridIcon,
+  Home as GridIcon,
```
Comment (leisterfrancisco) [inline webapp/src/routes/index.js:4]: shouldn't we rename it as HomeIcon?

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by tetogomez — APPROVED** [review]
LGTM

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**